addons/mrp/__init__.py

```diff
@@ -12,9 +12,7 @@ def _pre_init_mrp(env):
         - Creating the computed+stored field stock_move.unit_factor is terribly
         slow with the ORM and leads to "Out of Memory" crashes
     """
-    env.cr.execute("""ALTER TABLE "stock_move" ADD COLUMN "unit_factor" double precision;""")
-    env.cr.execute("""UPDATE stock_move
-                     SET unit_factor=1;""")
+    env.cr.execute("""ALTER TABLE "stock_move" ADD COLUMN "unit_factor" double precision NOT NULL DEFAULT 1;""")
```

Comment:
Do we want to keep that `default 1` afterwards or should it be removed once `1` has been set everywhere?